amundsen_application/static/js/config/config-utils.ts

```diff
@@ -90,6 +90,16 @@ export function getBadgeConfig(badgeName: string): BadgeStyleConfig {
   };
 }
 
+/*
+ * maps badge type to a badge style
+ */
+export function mapBadgeStyle(badgeType: string): BadgeStyle {
```

Comment:
nit: This isn't particularly related to a configuration so I'd suggest `BadgeList` own this logic.